src/window/sdl/Cargo.toml

```diff
@@ -14,7 +14,7 @@
 
 [package]
 name = "gfx_window_sdl"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
I believe this should be "0.7.0", since the `sdl2` dependency gets exposed in the API